GeoLib/IO/FEFLOW/FEFLOWGeoInterface.h

```diff
@@ -0,0 +1,76 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef FEFLOWGEOINTERFACE_H_
+#define FEFLOWGEOINTERFACE_H_
+
+#include <iosfwd>
+#include <memory>
+#include <string>
+#include <vector>
+
+class QDomElement;
+class QString;
+
+namespace GeoLib
+{
+    class GEOObjects;
+    class Point;
+    class Polyline;
+}
+
+namespace GeoLib
+{
+namespace IO
+{
+
+/**
+ * Read FEFLOW model files (*.fem) into OGS data structure. Currently this class supports
+ * only import of mesh data and some geometry given in Supermesh section.
+ */
+class FEFLOWGeoInterface
+{
+public:
+    /**
+     * read a FEFLOW Model file (*.fem) in ASCII format (Version 5.4)
+     *
+     * This function reads mesh data in addition to geometry data given in Supermesh.
+     *
+     * @param filename  FEFLOW file name
+     * @return a pointer to a created OGS mesh
+     */
+    void readFEFLOWFile(const std::string &filename, GeoLib::GEOObjects& obj);
+
+private:
+    // CLASS
+    struct FEM_CLASS
```

Comment:
There is a similar struct in the `FEFLOWMeshInterface`. Since there are other strong similarities between `FEFLOWGeoInterface` and `FEFLOWMeshInterface` for me it seems that both classes should have a common base class `FEFLOWInterface`.